plugins/processors/t128_transform/t128_transform.go

```diff
@@ -163,6 +178,8 @@ func (r *T128Transform) Init() error {
 
 	switch r.Transform {
 	case "diff":
+		r.PersistTo = ""
```

Comment:
I think I'd make this an `Init` error if `PersistTo != ""` in these unsupported cases. Otherwise users may get confused why the persistence file isn't being created etc.